share/completions/pg_restore.fish

```diff
@@ -0,0 +1,47 @@
+# General options:
+complete -c pg_restore -s d -l dbname -x -a '(__fish_complete_pg_database)' -d "connect to database name"
+complete -c pg_restore -s f -l file -a '-' -r -d "output file name (- for stdout)"
+complete -c pg_restore -s F -l format -x -a "c d t" -d "backup file format (should be automatic)"
```

Comment:
From https://www.postgresql.org/docs/9.2/app-pgrestore.html:

```
c
custom

    The archive is in the custom format of pg_dump.
d
directory

    The archive is a directory archive.
t
tar

    The archive is a tar archive.
```

That information would be nice to have as a description:

```fish
complete -c pg_restore -s F -l format -x -a \
"c\t'Custom format' d\t'Directory archive' t\t'Tar archive'"
```

(and possibly also the long names)

Reply:
(same goes for pg_dump, of course)

Reply:
Thank you. `pg_dump` and `pg_restore` seem to ignore everything but the first letter, so I left out the long names.